app/controllers/claims_controller.rb

```diff
@@ -168,8 +168,10 @@ def prepend_view_path_for_policy
   def one_time_password
     case params[:slug]
     when "email-address"
-      ClaimMailer.email_verification(current_claim, otp.code).deliver_now if current_claim.valid?(:"email-address")
-      session[:sent_one_time_password_at] = Time.now
+      if current_claim.valid?(:"email-address")
```

Comment:
Suggestion: we could use `current_claim.validate!(:"email-address")` here instead and rescue the exception. Otherwise it will fail silently if there is a validation error.

Reply:
@slorek Same as the other comment, I wasn't going to touch this file as it has nothing to do with the reminders, but tidying up, really should be another PR to address the OTP in the same way for this controller to catch the Notify exception.